eg_staged_bib_overlay

```diff
@@ -393,35 +337,80 @@ sub handle_load_bibs {
     report_progress('Number of bibs to update', $max);
     for (my $i = 1; $i <= $max; $i++) {
         report_progress('... bibs updated', $i) if 0 == $i % 10 or $i == $max;
-        $dbh->begin_work;
-        $dbh->do(qq{
-            UPDATE biblio.record_entry a
-            SET marc = b.marc
-            FROM $schema.$batch b
-            WHERE a.id = b.bib_id
-            AND bib_id IN (
-                SELECT bib_id
-                FROM $schema.$batch
-                WHERE to_import
-                AND NOT imported
-                ORDER BY bib_id DESC
-                LIMIT 1
+
+        my $query = qq{
+            SELECT bib_id FROM $schema.$batch
+            WHERE to_import AND NOT imported
+            ORDER BY bib_id DESC LIMIT 1
+        };
+        my $getbibid = $dbh->prepare($query);
+        $getbibid->execute();
+        my $target_bib_id = $getbibid->fetchrow_arrayref()->[0];
+
+        $query = qq{
+            SELECT EXISTS(
+                SELECT id
+                FROM biblio.record_entry
+                WHERE deleted
+                AND id = $target_bib_id
             )
-        });
-        $dbh->do(qq{
-            UPDATE $schema.$batch
-            SET imported = TRUE
-            WHERE bib_id IN (
-                SELECT bib_id
+        };
+        my $getdeleted = $dbh->prepare($query);
+        $getdeleted->execute();
+        my $deleted = $getdeleted->fetchrow_arrayref()->[0];
+
+        $query = qq{
+            SELECT EXISTS(
+                SELECT id
+                FROM biblio.record_entry
+                WHERE NOT deleted AND edit_date >= ?
+                AND id = $target_bib_id
+            )
+        };
+        my $editedafter = $dbh->prepare($query);
+        $editedafter->execute($cutoff);
+        my $edited_after_cutoff = $editedafter->fetchrow_arrayref()->[0];
+
+        $query = qq{
+            SELECT EXISTS(
+                SELECT id
                 FROM $schema.$batch
-                WHERE to_import
-                AND NOT imported
-                ORDER BY bib_id DESC
-                LIMIT 1
+                WHERE xml_is_well_formed(marc) AND id = $target_bib_id
             )
-        });
+        };
+        my $notwellformed = $dbh->prepare($query);
+        $notwellformed->execute();
+        my $xml_not_well_formed = $notwellformed->fetchrow_arrayref()->[0];
+
+        my $skip_reason;
+        if ($deleted eq 1) { $skip_reason = 'deleted'; }
+        if ($edited_after_cutoff eq 1) { $skip_reason = 'edited after cutoff'; }
```

Comment:
Please make the skip reason for this condition match its previous value exactly.